gulpfile.js

```diff
@@ -22,27 +26,53 @@ var paths = {
  *
  ***************************************/
 
+// full list of rules at: https://github.com/sasstools/sass-lint/tree/develop/docs/rules
+gulp.task('lint', function() {
+    return gulp.src(paths.src.sass + '**/*.s+(a|c)ss')
+        .pipe(sasslint({
+            options: {
+                formatter: 'stylish',
+                'merge-default-rules': true,
+            },
+            files: {ignore: paths.src.sass + 'vendor/*.scss'},
+            rules: {
+                'no-ids': 2,
```

Comment:
Do we plan to use ID for CLF elements? If so, will this rule always show an error?

Reply:
My personal feeling is we shouldn't allow them, especially for a framework. They are, by design, unique. In a specific implementation they may make sense or even be necessary, but they shouldn't be encouraged.

Of course, that's my personal bias and I think we can all discuss these rules.

Reply:
Sure, make sense, maybe we can discuss on how to implement CLF bias but overall framework should not allow IDs.